src/lib/y2storage/volume_specification_builder.rb

```diff
@@ -156,7 +156,6 @@ def fallback_for_prep
         v.min_size = DiskSize.MiB(2)
         v.desired_size = DiskSize.MiB(4)
         v.max_size = DiskSize.MiB(8)
-        # So far we are always using msdos partition ids
```

Comment:
why primary is not here, but at boot requirements? I have to say it is a bit confusing to have it at two places. What is rationale?

Reply:
`VolumeSpecification` and `Planned::Partition` are in fact different beasts. The first one can represent both a LV and a partition. It's the OOP direct representation of the volumes as they are specified in the control file.

A `Planned::Partition` is a more down-to-earth representation of something that is indeed being to be created, and contains additional information that is only relevant for partitions.

Reply:
BTW, in the past we hadn't `Planned::Partition` and `Planned::LvmLv`, we did the whole process with a more general `PlannedVolume` class... and the code turned to be VERY hard to follow. 

Creating more concrete and specialized `Planned::Partition` and `Planned::LvmLv` objects helped a lot making understandable code.